api/search_test.go

```diff
@@ -14,6 +18,7 @@ const validQueryParam string = "a"
 const validQueryDoc string = `{"valid":"elastic search query"}`
 const validESResponse string = `{"raw":"response"}`
 const validTransformedResponse string = `{"transformed":"response"}`
+const internalServerErrMsg = "internal server error"
```

Comment:
Possibly collapse all constants together, like so:

```suggestion
const (
    ...
    internalServerErrMsg = "internal server error"
)
```